app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -146,11 +146,15 @@
             {{#if (eq field.type 'image')}}
               {{widgets/forms/image-upload
                 imageUrl=(mut (get data.speaker field.fieldIdentifier))
-                label=(t 'Logo')
+                needsCropper=true
```

Comment:
why?

Reply:
that is how it is implemented in wizard basic-details form

Reply:
Why is needsCropper=true added here?

Reply:
it's compulsory to add needsCropper == true if want to use cropper any where.